syntax = "proto2";

package prototest;

enum GoodsKind {
  kDefault = 0;
  kFood = 1;
  kTool = 2;
  kClothing = 3;
}

message GoodsItem {
  required uint64 id = 1;
  required string name = 2;
  required double price = 3;
  required GoodsKind kind = 4;
}
